state/earn/selectors.ts

```diff
@@ -1,12 +1,35 @@
 import { createSelector } from '@reduxjs/toolkit';
+import { wei } from '@synthetixio/wei';
 
 import { RootState } from 'state/store';
-import { toWei } from 'utils/formatters/number';
+import { truncateNumbers } from 'utils/formatters/number';
+
+const SECONDS_PER_DAY = 86400;
```

Comment:
Seems we already have this constant declared in a few places if you wanted to import instead

Reply:
Actually, we don't seem to have it declared anywhere.

Reply:
There's one in `sdk/constants/period`

Reply:
My bad. Was searching for `86400` instead.